ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh

```diff
@@ -301,6 +301,14 @@ if [[ "$T5CI_VERSION" == "4.15" ]] || [[ "$T5CI_VERSION" == "4.16" ]]; then
     # git version 1.8 doesn't work well with forked repositories, requires a specific branch to be set
     sed -i "s@https://github.com/openshift/metallb-operator.git@https://github.com/openshift/metallb-operator.git\n        branch = main@" .gitmodules
     make init-git-submodules
+    checkout_submodules(){
+      submodule_path=$(git remote get-url origin)
+      repo=$(echo "$submodule_path" | sed 's/.*\///' | sed 's/\.git//')
```

Comment:
As these vars($org / $repo) are used elsewhere in script as well, I think would be safer to define with local keyword to limit the scope of vars.